relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__both_feature_flags_enabled.snap

```diff
@@ -178,6 +178,7 @@ expression: metrics
             "transaction": "gEt /api/:version/users/",
             "transaction.method": "POST",
             "transaction.op": "myop",
+            "user.geo.subregion": "155",
```

Comment:
Something to note here, this was added here because we added the browser context to the test event, if the browser context was missing, these tags would not be added to any spans.